controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -98,7 +97,7 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   virtual return_type update(const rclcpp::Time & time, const rclcpp::Duration & period) = 0;
 
   CONTROLLER_INTERFACE_PUBLIC
-  std::shared_ptr<rclcpp::Node> get_node();
+  std::shared_ptr<rclcpp_lifecycle::LifecycleNode> get_lifecycle_node();
```

Comment:
I'd really prefer to only change the return type of this, not the name.

Reply:
Sure thing, I will change it back.